CSV_Modifier_Client/Controllers/CsvUpload.cs

```diff
@@ -0,0 +1,60 @@
+﻿using Amazon.S3.Transfer;
+using Amazon.S3;
+using CSV_Modifier_Client.Models;
+using Microsoft.AspNetCore.Mvc;
+using Amazon;
+using CSV_Modifier_Client.Services;
+
+namespace CSV_Modifier_Client.Controllers
+{
+    public class CsvUpload : Controller
+    {
+        private readonly string bucketName = "csv-files-s3-bucket";
```

Comment:
move this to some constant file and use where ever required. it is being used on multiple places.